ANT+_Android_SDK/Sample/ANT+ Plugin Sampler/source/src/main/java/com/dsi/ant/antplus/pluginsampler/pulsezones/Dialog_DataInput.java

```diff
@@ -22,18 +24,23 @@
 public class Dialog_DataInput extends  DialogFragment {
 
     RadioGroup radioSexGroup;
-    EditText et_age;
-    EditText et_rest_hr;
+    ValidatableEditText et_age;
+    ValidatableEditText et_rest_hr;
     EditText et_max_hr;
     RadioGroup radioZones;
+    SharedPreferences myPrefs;
 
     int selectedIdSex = 0;
     int selectedIdZone = 1;
 
     @Override
     public Dialog onCreateDialog(Bundle savedInstanceState) {
         AlertDialog.Builder builder = new AlertDialog.Builder(getActivity())
-                .setTitle("Settings Configuration");
+                .setTitle("Settings Configuration")
```

Comment:
You can exercise and use Resources to externalize the labels, titles etc.
In this case, you will allow support of multiple languages and one day when we publish it in the Playstore ... you may have more users than just in English.

Reply:
done